src/themes/alex-party.css

```diff
@@ -0,0 +1,94 @@
+:root {
+	--twitch-gray: rgb(70, 70, 95);
+	--twitch-light: rgb(239, 239, 241);
+	--twitch-dark-background: rgb(24, 24, 27);
+	--twitch-purple: rgb(180, 84, 255);
+	--alex-border-color: blue;
+}
+
+html, body {
+	overflow: hidden;
+	height: 100vh;
+	margin: 0;
+}
+
+*{
+	box-sizing: border-box;
+}
+
+body {
+	background-color: transparent;
+}
+
+[data-twitch-chat] {
+	background-color: var(--twitch-light);
+	color: var(--twitch-dark-background);
+	border: 1em var(--alex-border-color) ridge;
+	border-radius: 1em;
+	font-family: "Comic Sans MS", cursive;
+	font-size: 24px;
+	list-style: none;
+	margin: 0;
+	padding: 0;
+
+	height: 100vh;
+	overflow: auto;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+}
+
+[data-twitch-message] {
+	line-height: 1.25em;
+	margin: 0;
+	padding-bottom: 0.25em;
+	padding-left: 0.5em;
+	padding-right: 0.5em;
+	padding-top: 0.25em;
+}
+
+[data-twitch-message-reply] {
+	color: var(--twitch-gray);
+	display: block;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+    overflow: hidden;
+	font-family: 'Papyrus', 'Comic Sans MS', cursive;
+	font-weight: 600;
+}
+
+[data-twitch-message] .twitch-chat-sender {
+	color: var(--twitch-sender-color, var(--twitch-purple));
+	display: inline;
+	font-weight: 700;
+	-webkit-text-stroke: 1px var(--alex-border-color);
+    text-shadow: 2px 2px 0px var(--alex-border-color);
+}
+
+[data-twitch-message] .twitch-chat-sender:after {
+	content: ": ";
+	color: var(--twitch-dark-background);
+	font-weight: normal;
+}
+
+[data-twitch-message] .twitch-chat-message {
+	display: inline;
+}
+
+[data-twitch-emote] {
+	width: 1.25em;
+}
+
+[data-twitch-mentioned-user] {
+	background-color: transparent;
+	font-family: 'Papyrus', 'Comic Sans MS', cursive;
+	font-weight: 600;
+	/* -webkit-text-stroke: 1px var(--twitch-purple); */
```

Comment:
Would you be willing to nuke this comment?

Reply:
killed it.